src/tagoio_sdk/modules/Account/Dashboards.py

```diff
@@ -0,0 +1,231 @@
+from typing import Optional, TypedDict
+from tagoio_sdk.common.Common_Type import ExpireTimeOption, GenericID
+from tagoio_sdk.common.tagoio_module import TagoIOModule
+from tagoio_sdk.modules.Account.Dashboards_Type import (
+    AnalysisRelated,
+    DashboardCreateInfo,
+    DashboardInfo,
+    DashboardQuery,
+    DevicesRelated,
+    PublicKeyResponse,
+)
+from tagoio_sdk.modules.Utils.dateParser import dateParser, dateParserList
+
+
+class Dashboards(TagoIOModule):
+    def listDashboard(self, queryObj: DashboardQuery = {}) -> list[DashboardInfo]:
+        """
+        Retrieves a list with all dashboards from the account
+
+        :default:
+
+            queryObj: {
+                "page": 1,
+                "fields": ["id", "name"],
+                "filter": {},
+                "amount": 20,
+                "orderBy": "label,asc",
+            }
+
+        :param queryObj Search query params
+        """
+
+        if "orderBy" in queryObj:
+            firstArgument = queryObj["orderBy"][0]
+            seccondArgument = queryObj["orderBy"][1]
+            orderBy = f"{firstArgument},{seccondArgument}"
+        else:
+            orderBy = "label,asc"
+
+        result = self.doRequest(
+            {
+                "path": "/dashboard",
+                "method": "GET",
+                "params": {
+                    "page": queryObj.get("page") or 1,
+                    "fields": queryObj.get("fields") or ["id", "name"],
+                    "filter": queryObj.get("filter") or {},
+                    "amount": queryObj.get("amount") or 20,
+                    "orderBy": orderBy,
+                },
+            }
+        )
+
+        result = dateParserList(result, ["created_at", "updated_at", "last_access"])
+        return result
+
+    class dashboard(TypedDict):
+        dashboard: GenericID
+
+    def create(self, dashboardObj: DashboardCreateInfo) -> dashboard:
+        """
+        Create dashboard
+
+        :param dashboardObj Dashboard object
+        """
+        result = self.doRequest(
+            {
+                "path": "/dashboard",
+                "method": "POST",
+                "body": {dashboardObj},
```

Comment:
@rafaelTelesS you can remove the dictionary if not it will get a TypeError: unhashable type: 'dict'
```python
result = self.doRequest(
            {
                "path": "/dashboard",
                "method": "POST",
                "body": {dashboardObj},
            }
        )
```